README.md

```diff
@@ -201,7 +208,7 @@ The support of Turbolinks was dropped in [d8c7d9f](https://github.com/decidim/de
 
 ## Following our license
 
-If you plan to release your application you'll need to publish it using the same license: GPL Affero 3. We recommend doing that on Github before publishing, you can read more on "[Being Open Source From Day One is Especially Important for Government Projects](http://producingoss.com/en/governments-and-open-source.html#starting-open-for-govs)". If you have any trouble doing that you can contact us on [Gitter](https://gitter.im/decidim/decidim).
+If you plan to release your application you'll need to publish it using the same license: GPL Affero 3. We recommend doing that on Github before publishing, you can read more on "[Being Open Source From Day One is Especially Important for Government Projects](http://producingoss.com/en/governments-and-open-source.html#starting-open-for-govs)". If you have any trouble doing that you can contact us on [What's the best way to contact you?]().
```

Comment:
Commit removed, hope you don't mind force-pushing (it makes me sad adding a change and reverting it in the same PR).